Services/AccessControl/classes/class.ilRbacAdmin.php

```diff
@@ -263,11 +263,8 @@ protected function addDesktopItem($a_rol_id, $a_usr_id)
 
 	/**
 	 * Assigns an user to a role. Update of table rbac_ua
-	 * TODO: remove deprecated 3rd parameter sometime
-	 * @access	public
-	 * @param	integer	object_id of role
-	 * @param	integer	object_id of user
-	 * @param	boolean	true means default role (optional
+	 * @param	int	$a_rol_id Object-ID of role
```

Comment:
Only some boyscouting, the 3rd parameter has already been removed, only the PHPDoc was wrong.